src/Models/class-smart-link.php

```diff
@@ -39,6 +39,16 @@ class Smart_Link extends Base_Model {
 	 */
 	public $source_post_id = 0;
 
+	/**
+	 * The context of the smart link.
+	 *
+	 * For example, 'traffic_boost' or 'smart_linking'.
+	 *
+	 * @since 3.18.0
+	 * @var string|false The context of the smart link.
```

Comment:
Did we consider using `null` instead of `false`? Do we think `false` is better?

Reply:
Yeah, probably that makes more sense. Will replace!